packages/cqrs/lib/src/cqrs.dart

```diff
@@ -125,6 +125,39 @@ class Cqrs {
     );
   }
 
+  /// Send a operation to the backend and expect a result of the type `T`.
+  ///
+  /// Headers provided in `headers` are on top of the `headers` from the [Cqrs]
+  /// constructor, meaning `headers` override `_headers`. `Content-Type` header
+  /// will be ignored.
+  ///
+  /// A [CqrsException] will be thrown in case of an error.
+  Future<T> perform<T>(
```

Comment:
I won't think I can help because we have completelly different model. We just invoke the specific method. So if operation is for example called `EditUser` we just have `api.editUser({ id: "" })`.

Reply:
OK then, `perform` is fine for me!